lib/ansible/modules/cloud/kubevirt/kubevirt_rs.py

```diff
@@ -202,10 +180,17 @@ def execute_module(self):
         changed = result_crud['changed']
         result = result_crud.pop('result')
 
-        # Wait for the replicas:
-        wait = self.params.get('wait')
-        if wait:
-            result = self.wait_for_replicas()
+        # When creating a new VMIRS object without specifying `replicas`, assume it's '1' to make the
+        # wait logic work correctly
+        if changed and result_crud['method'] == 'create' and replicas is None:
+            replicas = 1
+
+        # Wait for the new number of ready replicas after a CRUD update
+        # Note1: doesn't work correctly when reducing number of replicas due to how VMIRS works (as of kubevirt 1.5.0)
```

Comment:
Let's create a story to fix it once improved on kubevirt side.

Reply:
Quoting Roman Mohr: _I can't remember if that was intentional (even have a test for it)_. So this might be intended, might not, might be revisited in a few months, might not, no way to tell at this point. If this module gets used down the line, someone might revisit this, which is why I've added the comment.

Reply:
Though note2 might warrant a story…